js/views/currencyListVw.js

```diff
@@ -45,7 +46,6 @@ module.exports = Backbone.View.extend({
     },this);
 
     this.$el.append('<ul class="flexRow list homeModal-settings scrollOverflowY custCol-primary custCol-text customThemeScrollbar">'+ this.listContents.join('') +'</ul>');
```

Comment:
Clearing the $el content with this.$el.html(''); before this line or using html() instead of append() fixes the double render bug, and it's probably a good idea to clear the $el regardless, but it would be better to fix whatever is causing getBTPrice to fire twice.